python/tests/parquet/test_dataset.py

```diff
@@ -30,11 +31,14 @@ def test_select_columns(spark: SparkSession, tmp_path: Path):
             Row(id=2, col1="more", col2=456),
         ]
     )
-    df.write.format("rikai").save(str(tmp_path))
+    rikai_path = str(tmp_path / "test_select_columns")
+    df.write.format("rikai").save(rikai_path)
 
-    dataset = Dataset(str(tmp_path), columns=["id", "col1"])
+    dataset = Dataset(rikai_path, columns=["id", "col1"])
     actual = sorted(list(dataset), key=lambda x: x["id"])
 
     assert_count_equal(
         [{"id": 1, "col1": "value"}, {"id": 2, "col1": "more"}], actual
     )
+
+    shutil.rmtree(rikai_path)
```

Comment:
`tmp_path` will be automatically deleted by `pytest` framework. we dont need to manually `rmtree`.

https://docs.pytest.org/en/stable/reference.html#std-fixture-tmp_path